webapp/src/app/pacman-features/modules/admin/user-management/user-management.component.ts

```diff
@@ -178,7 +178,6 @@ export class UserManagementComponent implements OnInit, AfterViewInit {
         }
       }catch(error){
         this.errorMessage = this.errorHandling.handleJavascriptError(error);
```

Comment:
Why keep this.errorMessage when it's just being set if it's not used?

Reply:
> Why keep this.errorMessage when it's just being set if it's not used?

yeah. We don't need to set here.